### DISCUSSION TITLE
.htaccess for YOURLS at root with existing site in subdirectory

### DISCUSSION THREAD

**Discussion by k1t1q** [YOURLS/YOURLS#3542] [Q&A]
I have a forum at domain.com
i want to install yourls to the root of domain.com
it is important that when the user enters the address domen.com, he is redirected to domen.com, as it is now
how do i edit .htaccess to make both forum and yourls work?
```
now without yourls, .htaccess looks like this
<IfModule mod_rewrite.c>
Options -MultiViews
RewriteEngine On
RewriteBase /forum/
RewriteCond %{REQUEST_FILENAME} !-f
RewriteRule \.(js|css|jpeg|jpg|gif|png|ico|map)(\?|$) /forum/404error.php [L,NC]

RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule . /forum/index.php [L]
</IfModule> 
```

**Comment by ozh** [comment]
I wouldn't do much with the .htaccess
- You have domain.com/forum
- Install YOURLS at the root, with nothing special done
- Make an index.php with simply a redirect to /forum
- Use a plugin to handle short URLs not found (to avoid being simply redirected to /forum)

**Reply by k1t1q** [reply]
> I wouldn't do much with the .htaccess
> 
> * You have domain.com/forum
> * Install YOURLS at the root, with nothing special done
> * Make an index.php with simply a redirect to /forum
> * Use a plugin to handle short URLs not found (to avoid being simply redirected to /forum)

The forum and the short URLs are located at the base folder directory domain.com. While .htaccess1  is used only the forum works, while .htaccess2 - only short URLs. Is there a way to combine two files so they work fine together?

**Reply by dgw** [reply]
> Is there a way to combine two files so they work fine together?

From YOURLS server configuration [instructions](https://yourls.org/docs/guide/server-configuration#make-a-htaccess-file):

> Note: YOURLS **cannot share** its root directory with another `.htaccess` rewrite rules driven app such as WordPress. The `.htaccess` file should contain YOURLS directives and no other rewrite rule.
